tools/vegan_rarefaction/vegan_rarefaction.xml

```diff
@@ -0,0 +1,114 @@
+        <param name="sample_size" type="integer" value="" min="1" optional="True" label="Subsample size for rarefying community" help="leave empty to subsample by community size for the smallest sample"/>
+        <!-- <param name="estimate_standard_error" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="False" label="Estimate standard errors"/> -->
+        <!-- <param name="margin" type="integer" value="1" label="Margin for which the index is computed" /> -->
+        
+        <param name="step_size" type="integer" value="1" min="1" label="Step size for sample sizes"/>
+        <param name="xlab" type="text" value="Sample Size" label="X-axis label"/>
+        <param name="ylab" type="text" value="Species" label="Y-axis label"/>
+        <param name="label" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="True" label="Label rarefaction curves by rownames of X"/>
+
+        
+    </inputs>
+    <outputs>
+        <data format="tabular" name="output_species_count" label="${tool.name} on ${on_string} (number of species)"/>
+        <data format="tabular" name="output_species_frequency" label="${tool.name} on ${on_string} (frequency of species)"/>
+        <data format="tabular" name="output_richness" label="${tool.name} on ${on_string} (estimated richness)"/>
+        <data format="tabular" name="output_species_probabilities" label="${tool.name} on ${on_string} (species probabilities)"/>
+        <data format="tabular" name="output_slope" label="${tool.name} on ${on_string} (slope of curve)"/>
+        <data format="txt" name="vegan_rarefaction_script_output" label="${tool.name} on ${on_string} (Rscript)" />
+        <data format="png" name="output_plot" label="${tool.name} on ${on_string} (plot)" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_abundance" ftype="tabular" value="vegan_in.tabular"/>
+            <param name="species_column" value="6"/>
+            <param name="sample_columns" value="2"/>
+            <param name="sample_size" value=""/>
+            <param name="step_size" value="1"/>
+            <param name="xlab" value="Sample Size"/>
+            <param name="ylab" value="Species"/>
+            <param name="label" value="TRUE"/>
+            <output name="output_species_count" ftype="tabular" file="vegan_output_species_count.tabular" />
+            <output name="output_species_frequency" ftype="tabular" file="vegan_output_species_frequency.tabular" />
+            <output name="output_richness" ftype="tabular" file="vegan_output_richness.tabular" />
+            <output name="output_species_probabilities" ftype="tabular" file="vegan_output_species_probabilities.tabular" />
+            <output name="output_slope" ftype="tabular" file="vegan_output_slope.tabular" />
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+        
```

Comment:
This should be improved, at least a small description of the tool should be added.
An example would be also nice.